Daniel-Bennett777/Buildo-API
# finished registration and login

edited our `users.py` to include a rare_user instance in our tables.

**To test**
git checkout main
git fetch
git checkout DP
now go to postman and make request to `register/` url make sure your format body is including all user and rareuser fields like such :  `{
    "username": "charles123",
    "email": "htkbaby@gmail.com",
    "first_name": "paul",
    "last_name": "brown",
    "password": "kass34",
    "bio": "never lose. if you aint first your last ",
    "profile_image_url": "https://images.pexels.com/photos/762080/pexels-photo-762080.jpeg?auto=compress&cs=tinysrgb&w=600",
    "active": true,
    "state_name": "Tennessee",
    "county_name": "Davidson",
    "is_contractor": true
}`   add your own fields to register a user. also you can check the login at `login/`
